content/docs/2.8/providers/aws-eks.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "EKS Pod Identity Webhook for AWS"
+layout = "provider"
```

Comment:
Drop `layout = "provider"` from this and other pages since the layout isn't being picked up. In any case, there is nothing special about how this page is rendered, right?

Reply:
Not, nothing. Removed!